products/redis/api.yaml

```diff
@@ -96,7 +96,10 @@ objects:
         value_type: Api::Type::String
       - !ruby/object:Api::Type::NameValues
         name: 'redisConfigs'
-        description: Redis configuration parameters, according to http://redis.io/topics/config.
+        description: |
+          Redis configuration parameters, according to http://redis.io/topics/config.
+          Currently, the only supported parameters are maxmemory-policy and notify-keyspace-events
```

Comment:
@Chupaka thanks for contributing!

Rather then list the supported parameters here I would just point to the url you provider and say "Check the google documentation for the list of supported parameters". Pointing user's to the source of truth for "currently supported blah" will mean that when those documents are updated then ours won't be wrong and it will reduce churn in this repo.

Reply:
Done. That was my thought initially, but then I decided that the list of two parameters does not worth clicking the link and waiting for the whole new page loading...